.github/workflows/build.yml

```
@@ -2,126 +2,89 @@ name: Build
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
 
 jobs:
   buildAndCheck:
     strategy:
       fail-fast: false
       matrix:
-        java: [ '23', '17' ]
-        os: [ 'macos', 'windows', 'ubuntu' ]
+        os: [ 'ubuntu-24.04', 'macos-15', 'windows-2025' ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)